server/Controllers/PointsController.cs

```diff
@@ -6,21 +6,26 @@
 using hutel.Models;
 using Microsoft.AspNetCore.Mvc;
 using Microsoft.Extensions.Caching.Memory;
+using Microsoft.Extensions.Logging;
 using Newtonsoft.Json;
 
 namespace hutel.Controllers
 {
     [Route("api/[controller]")]
     public class PointsController : Controller
     {
-        private IMemoryCache _memoryCache;
+        private readonly IMemoryCache _memoryCache;
+        private readonly ILogger _logger;
         private const string _pointsKey = "points";
         private const string _tagsKey = "tags";
-        private const string _storagePath = ".\\server\\storage.json";
-        private const string _storageBackupPath = ".\\server\\storage.json.bak";
-        private const string _tagsPath = ".\\server\\tags.json";
+        private static readonly string _storagePath =
+            System.IO.Path.Combine(".", "server", "storage.json");
```

Comment:
I do:
 cd ./server
dotnet build
dotnet run

in this configuration my relative pathes are  System.IO.Path.Combine(".", "storage.json") ; System.IO.Path.Combine(".", "storage.json.bak");  System.IO.Path.Combine(".", "tags.json");

So I mean they must not have "server" part in the relative paths.